Features

maxorax opened the pull request:
Launch XiB

maxorax committed:
update viewController

maxorax committed:
launch xib

anzorin1996 commented on ContactList/AppDelegate.swift:
```diff
@@ -13,8 +13,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        self.window = UIWindow(frame: UIScreen.main.bounds)
+        print("width = \(UIScreen.main.bounds.width)")
```
отладочную информацию нужно удалять - у нас main это основная ветка, где все должно быть красиво. в своих ветках можешь делать все, что угодно

anzorin1996 commented on ContactList/AppDelegate.swift:
```diff
@@ -13,8 +13,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
+        let vc = ViewController()
+        
+        self.window?.rootViewController = vc
+        UIView.transition(
```
это тоже удали

maxorax committed:
Добавил кнопкну для аутенфикации на RegisterLoginViewController.
Создал новый контроллер с xib ContactViewController и добавил туда кнопку logout.

maxorax committed:
Update AppDelegate.swift

Убрал отладочную информацию

maxorax committed:
Create AppDelegate.swift

Убрал отладочную информацию

maxorax committed:
Delete Pods directory

maxorax committed:
update

maxorax committed:
Убрал логику из appDelegate и перенес ее в RootViewController

maxorax committed:
исправил переходы

maxorax committed:
Поправил переходы
